nibabel/freesurfer/io.py

```diff
@@ -129,6 +181,22 @@ def write_geometry(filepath, coords, faces, create_stamp=None):
         coords.astype('>f4').reshape(-1).tofile(fobj)
         faces.astype('>i4').reshape(-1).tofile(fobj)
 
+        # Add volume info, if given
+        if volume_info is None or len(volume_info) == 0:
+            return
+
+        for key, val in volume_info.items():
+            if key == 'head':
```

Comment:
should we make sure that the first key is `head` and that it's the value that we expect?